newspack-theme/woocommerce/templates.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * This folder holds Woocommerce templates that override the default Woo templates.
+ *
+ * If you add or remove a template from this folder, it's a good idea to clear the template cache
+ * to make sure the changes take effect - and not break anything - on sites that use persistent caching
+ *
+ * @package newspack-theme
+ */
+
+/**
+ * When adding or removing a template from this folder, increase the templates_version variable below.
+ */
+$newspack_theme_woo_templates_version = 1;
+
+if ( get_option( 'newspack_theme_woo_templates_version', 0 ) !== $newspack_theme_woo_templates_version ) {
```

Comment:
Edge case, but this ends up throwing a fatal if WooCommerce is deactivated unless we check for the existence of the function first:

```suggestion
if ( function_exists( 'wc_clear_template_cache' ) && get_option( 'newspack_theme_woo_templates_version', 0 ) !== $newspack_theme_woo_templates_version ) {
```

Reply:
Good call